### DISCUSSION TITLE
关于第三方资源管理器的支持

### DISCUSSION THREAD

**Discussion by JenryMa** [shuax/MouseInc.Settings#193] [General]
开发者好，
试用了Files, MyFiles-X等第三方资源管理器，都无法正常使用手势，请问有计划针对第三方资源管理器提供支持吗？ 或者设置里面增加什么选项可以做到支持？ 

多谢。

**Comment by shuax** [comment]
这些是不是UWP程序，UWP程序好像无法操纵
